SDK Showcase/SDK Showcase/Interactors/SDKInteractor.swift

```diff
@@ -58,23 +53,18 @@ class SDKInteractorReal: SDKInteractor {
         appState.registeredUsers
             .forEach { userData in
                 userData.authenticatorsNames.forEach { authenticatorName in
-                    toReturn.append(formatCategoryName(userId: userData.userId, authenticatorName: authenticatorName))
```

Comment:
why to "flatten" the logic here to recreate in some way "When more authenticators added in next PRs"? Do you have the logic with "more authenticators" already on the side to push it right away after this PR?